demo-cli/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+group = rootProject.group
+version = rootProject.version
+
+dependencies {
+    implementation("com.github.ajalt:clikt:2.8.0")
+    implementation("com.google.code.gson:gson:2.7")
+}
+
+tasks {
+    runIde {
+        val input: String? by project
+        val output: String? by project
+        args = listOfNotNull("DemoPluginCLI", input, output)
```

Comment:
It would be nice to have the command name as a parameter in the `runIde` task (or the task that invokes it), rather than hardcoded: one might want to implement several commands within one plugin.

I have no idea what is the proper way to do it with gradle, but here is a (rather old) discussion about parameterized tasks: https://discuss.gradle.org/t/passing-arguments-to-a-task/8427

Reply:
Fixed it. Now it is possible to specify a command to run